docs/kratos/social-signin/83_linkedin.mdx

```diff
@@ -0,0 +1,145 @@
+---
+id: linkedin
+title: Add LinkedIn as a social sign-in provider in Ory
+sidebar_label: LinkedIn
+---
+
+# LinkedIn
+
+````mdx-code-block
+import JsonnetWarning from "../../_common/jsonnetwarning.mdx"
+import Tabs from '@theme/Tabs';
+import TabItem from '@theme/TabItem';
+
+<Tabs>
+<TabItem value="console" label="Ory Console" default>
+
+Follow these steps to add LinkedIn as a social sign-in provider to your project using the Ory Console:
+
+1. Sign in to Ory Console and select [**Social Sign-in**](https://console.ory.sh/projects/current/social-signin).
+2. Click the switch next to the LinkedIn logo to start the configuration.
+3. Copy the Redirect URI and save it for later use.
+4. [Create a LinkedIn application](https://developer.linkedin.com/), verify it and [activate **Sign In with LinkedInV2**](https://www.linkedin.com/pulse/how-get-signin-linkedin-work-taric-andrade/).
```

Comment:
```suggestion
4. [Create a LinkedIn application](https://developer.linkedin.com/), verify it and [activate **Sign In with LinkedInV2**](https://www.linkedin.com/pulse/how-get-signin-linkedin-work-taric-andrade/).
```

Where do you get the `V2` in the name from?

Reply:
That is what using OIDC is called at LinkedIn. 
They also have a v1 version that is using something different (oauth1) i think

Reply:
I don't see a way to create a `V1` app through their dev portal. Adding a version denominator isn't necessary. Please remove it.

Reply:
Sorry, not sure what you are looking at, but it is necessary to have this because there are 2 way to do "login with linkedin" 

v2 will work with Ory, the other one not.

![Screenshot 2023-03-20 at 12 29 56](https://user-images.githubusercontent.com/55394283/226327008-b42a29ac-ea23-4dcd-ab01-5d8c00c55695.png)